fusion/whitelist.json

```diff
@@ -1,6 +1,5 @@
 {
  "files": [
-  "fusion/whitelist.json",
```

Comment:
did we mean to remove this?

Reply:
I didn't remove this. I believe @zinigor did.

Reply:
Yes, he did intentionally, we don't want to check that file.